packages/@uppy/dashboard/src/components/FileItem/MetaErrorMessage.js

```diff
@@ -0,0 +1,29 @@
+const { h } = require('preact')
+
+module.exports = function renderMissingMetaFieldsError (props) {
+  const { file, toggleFileCard, i18n } = props
+  const { missingRequiredMetaFields } = file
+  if (!missingRequiredMetaFields) {
+    return null
+  }
+  const metaFieldsString = missingRequiredMetaFields.map(field => (
+    field.charAt(0).toUpperCase() + field.slice(1)
+  )).join(', ')
```

Comment:
Is there a way to get the `label` instead of the `name` here by any chance?

Reply:
Only through Dashboard options, right? Which is not ideal, as in theory Core is independent of Dashboard and doesn’t know about it.

Reply:
But as you are making the UI for the dashboard and not for core, would it be possible to match ids of `requiredMetaFields` to `metaFields` and then use the `name` property from the latter?